src/dscom.build/doc/ReadMe.md

```diff
@@ -0,0 +1,229 @@
+          ║               │              false               │                                                                                 │                             │                               │                             │                             │                       │                            │                                │                     │                     ║
+          ║               │<─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─│                                                                                 │                             │                               │                             │                             │                       │                            │                                │                     │                     ║
+          ╚═══════════════╪══════════════════════════════════╪═════════════════════════════════════════════════════════════════════════════════╪═════════════════════════════╪═══════════════════════════════╪═════════════════════════════╪═════════════════════════════╪═══════════════════════╪════════════════════════════╪════════════════════════════════╪═════════════════════╪═════════════════════╝
+                    ┌─────┴─────┐                       ┌────┴────┐                                                                     ┌──────┴──────┐          ┌───────────┴────────────┐          ┌───────┴────────┐          ┌─────────┴─────────┐          ┌────────┴────────┐          ┌───┴────┐          ┌────────────┴─────────────┐          ┌───────┴────────┐          ┌─┴─┐                    
+                    │BuildEngine│                       │TlbExport│                                                                     │IBuildContext│          │TypeLibConverterSettings│          │FileSystemChecks│          │AssemblyLoadContext│          │AppDomain.Current│          │Assembly│          │LoggingTypeLibExporterSink│          │TypeLibConverter│          │Log│                    
+                    └───────────┘                       └─────────┘                                                                     └─────────────┘          └────────────────────────┘          └────────────────┘          └───────────────────┘          └─────────────────┘          └────────┘          └──────────────────────────┘          └────────────────┘          └───┘                    
+```
+
+## Test
+
+### Unit tests
+
+The class `TlbExport` is written with a level of abstraction and hence is tested with a [unit test](../../src/dscom.test/tests/BuildTaskTest.cs).
+
+### Acceptance tests
+
+The examples [32-bit](../../examples/32bit/) and [outproc](../../examples/outproc/) each provide an `msbuild-acceptance-test.cmd` script. This file builds and packages the build task and adds it to the `.csproj` file of the resulting assembly. Then the build is conducted. It must be warning free.

```

Comment:
Broken link. I won't write a comment on all the other following links, but please check that the linked files and folders exist